<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ManageAccount</class>
 <widget class="QDialog" name="ManageAccount" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>161</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="3" >
    <widget class="QTreeWidget" name="accountList" >
     <column>
      <property name="text" >
       <string>Login</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Protocol</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QToolButton" name="addButton" >
     <property name="text" >
      <string>+</string>
     </property>
     <property name="toolButtonStyle" >
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="arrowType" >
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QToolButton" name="removeButton" >
     <property name="text" >
      <string>-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
